contracts/proposal/dao-proposal-single/src/contract.rs

```diff
@@ -157,25 +152,28 @@ pub fn execute(
 pub fn execute_propose(
     deps: DepsMut,
     env: Env,
-    sender: Addr,
-    title: String,
-    description: String,
-    msgs: Vec<CosmosMsg<Empty>>,
-    proposer: Option<String>,
+    info: MessageInfo,
```

Comment:
any reason for switching from `sender: Addr` to `info: MessageInfo` here?

Reply:
nope, i think i did that before i realized the vote sender needs to be the proposer, and never undid it. will fix! nice catch.